packages/react-component-playground/test/components/component-playground/selected-fixture-and-editor/events/handlers.js

```diff
@@ -62,13 +62,11 @@ describe(`ComponentPlayground (${FIXTURE}) Events Handlers`, function() {
     });
   });
 
-  it('should set splitpane onChange to localstorage lib set', function() {
-    expect(component.refs.splitPane.props.onChange())
-      .to.equal(localStorageLib.set('splitPos'));
-  });
+  it('should have split-pane position in local storage on change', function() {
+    var size = 10;
 
-  it('should set splitpane defaultSize to localstorage lib get', function() {
```

Comment:
@bogdan-stefan-hs why did you remove this test? Is it not relevant to test that the defaultSizes is ready from the local storage?

I was just referring to humanising the test case strings in my previous comment.